pages/_next-prefetcher.js

```diff
@@ -0,0 +1,108 @@
+/* global self */
```

Comment:
@nkzawa anyway, I managed to reduced the bundle size by using `babel-preset-env` and not using async await.